### PR TITLE
Updated docs of repo

### PR DESCRIPTION
**Pull request by DK070202**
- Added detailed docs of repo.

### CODE REVIEW

**Review by princebansal — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -1,4 +1,375 @@
-# GrowthBook SDK Flutter.
+![](https://docs.growthbook.io/images/hero.png)
 
-GrowthBook SDK for flutter.
+# Growth Book - SDK
```
Comment (Auz) [inline README.md:3]: 1. GrowthBook has no space. (this is through out this document)
2. Removing the flutter from the name - is that on purpose? Just wondering if people will know what it's for or how it is different than the other SDKs.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,375 @@
...
+
+```yaml
+growthbook_sdk_flutter: ^1.0.0
+```
+
+## Integration
+
+Integration is super easy:
+
+1. Create a Growth Book API key
```
Comment (Auz) [inline README.md:32]: 1. Create a GrowthBook API key from the GrowthBook App

File: README.md [hunk 3]
```diff
@@ -1,4 +1,375 @@
-# GrowthBook SDK Flutter.
+![](https://docs.growthbook.io/images/hero.png)
 
-GrowthBook SDK for flutter.
+# Growth Book - SDK
 
+![](https://camo.githubusercontent.com/b1d9ad56ab51c4ad1417e9a5ad2a8fe63bcc4755e584ec7defef83755c23f923/687474703a2f2f696d672e736869656c64732e696f2f62616467652f706c6174666f726d2d616e64726f69642d3645444238442e7376673f7374796c653d666c6174) ![](https://camo.githubusercontent.com/1fec6f0d044c5e1d73656bfceed9a78fd4121b17e82a2705d2a47f6fd1f0e3e5/687474703a2f2f696d672e736869656c64732e696f2f62616467652f706c6174666f726d2d696f732d4344434443442e7376673f7374796c653d666c6174) ![](https://camo.githubusercontent.com/4ac08d7fb1bcb8ef26388cd2bf53b49626e1ab7cbda581162a946dd43e6a2726/687474703a2f2f696d672e736869656c64732e696f2f62616467652f706c6174666f726d2d74766f732d3830383038302e7376673f7374796c653d666c6174) ![](https://camo.githubusercontent.com/135dbadae40f9cabe7a3a040f9380fb485cff36c90909f3c1ae36b81c304426b/687474703a2f2f696d672e736869656c64732e696f2f62616467652f706c6174666f726d2d77617463686f732d4330433043302e7376673f7374796c653d666c6174)
+
+
+
```
Comment (Auz) [inline README.md:8]: Maybe add a brief overview here? Perhaps something like:

GrowthBook is an open source feature flagging and experimentation platform that makes it easy to adjust what features are shown users, and run A/B tests, without deploying new code. There are two parts to GrowthBook, the GrowthBook Application, and the SDKs which implement this functionality to your code base. This Flutter SDK allows you to use GrowthBook with your Flutter based mobile application.

File: README.md [hunk 4]
```diff
@@ -1,4 +1,381 @@
...
+```yaml
+growthbook_sdk_flutter: ^1.0.0
+```
+
+## Integration
+
+Integration is super easy:
+
+1. Create a GrowthBook API key from the GrowthBook App.
+2. At the start of your app,  do SDK Initialization as per below
```
Comment (Auz) [inline README.md:39]: Minor text edit: 
Initialize the SDK within your app using the API key, as below. 
or 
Initialize the SDK at the start of your app using the API key, as below.

**Pull request merged**